src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/client/ProfessionalApiClient.java

```diff
@@ -308,8 +325,12 @@ public void retrieveBadRequestForPendingOrganisationWithPbaEmail(String email, S
         response.then()
                 .assertThat()
                 .statusCode(status.value());
+        if (HttpStatus.OK == status) {
 
-        return response.body().as(Map.class);
+            return response.body().as(Map.class);
+        } else {

```

Comment:
You DON'T need an else here please remove (or alternatively, use a ternary operator instead with a single return statement)

Reply:
Removed else and added only single return statement